hist/hist/src/TH1.cxx

```diff
@@ -6654,7 +6654,7 @@ UInt_t TH1::SetCanExtend(UInt_t extendBitMask)
 }
 
 ///////////////////////////////////////////////////////////////////////////////
-/// Internal function used in TH1::Fill to see which axis is full alphanumeric
+/// Internal function used in TH1::Fill to see which axis is full alphanumeric.
```

Comment:
The following line seems to be part of this sentence, thus
```suggestion
/// Internal function used in TH1::Fill to see which axis is full alphanumeric,
```

In any case and out of curiosity, what is the rationale for also requiring that the axis can be extended?